integration/app/test-demo/src/pages/VideoFilterTestApp.tsx

```diff
@@ -0,0 +1,115 @@
+  VideoTileGrid,
+  FormField,
+  Select,
+  VideoInputBackgroundBlurControl,
+} from 'amazon-chime-sdk-component-library-react';
+import MeetingInfo from '../components/MeetingInfo';
+import MeetingLeaveControl from '../components/MeetingLeaveControl';
+import MeetingForm from '../containers/MeetingForm';
+
+export const VideoFilterTestApp: React.FC = () => {
+  const defaultOptions: BackgroundBlurOptions = {
+    blurStrength: 20,
+    filterCPUUtilization: 20,
+    reportingPeriodMillis: 1000,
+  };
+  const [backgroundBlurOptions, setBackgroundBlurOptions] = useState<BackgroundBlurOptions>(defaultOptions);
+
+  const setBlurStrength = (newBlurStrength: number): void => {
+    console.log(`Setting blur strength to ${newBlurStrength} - should re-initialize the bg blur provider`);
+    const options = {...backgroundBlurOptions, blurStrength : newBlurStrength};
+    setBackgroundBlurOptions(options);
+  }
+
+  return(
+    <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+      <BackgroundBlurProvider options={backgroundBlurOptions}>
+        <MeetingProvider {...{logLevel: LogLevel.INFO}}>
+          <h3 data-testid='app-name'>Video Filter Test</h3>
+          <Meeting onBlurStrengthChanged={(newBlurStrength) => setBlurStrength(newBlurStrength)}/>
+        </MeetingProvider>
+      </BackgroundBlurProvider>
+    </div>
+    )
+};
+
+interface Props {
+  onBlurStrengthChanged: (blurStrength: number) => void;
+}
+
+const Meeting: React.FC<Props> = ({ onBlurStrengthChanged }) => {
+  const { isBackgroundBlurSupported, createBackgroundBlurDevice } = useBackgroundBlur();
+  const meetingManager = useMeetingManager();
+  const { selectedDevice } = useVideoInputs();
+  const { toggleVideo } = useLocalVideo();
+  const meetingStatus = useMeetingStatus();
+  const [ blurStrength, setBlurStrength ] = useState<string>("20");
+
+  useEffect(() => {
+    const addBackgroundBlur = async () => {
+      if (
+        isBackgroundBlurSupported &&
+        meetingStatus === MeetingStatus.Succeeded
+      ) {
+        console.log("Creating background blur device called");
+        const chosenVideoTransformDevice = await createBackgroundBlurDevice(selectedDevice);
+        console.log(chosenVideoTransformDevice);
+        await meetingManager.selectVideoInputDevice(chosenVideoTransformDevice);
+        toggleVideo();
+      }
+    };
+    addBackgroundBlur();
+  }, [
+    isBackgroundBlurSupported,
+    meetingStatus,
+  ]);
+
+  const updateBlurStrength = (e: ChangeEvent<HTMLSelectElement>) => {
+    e.preventDefault();
+    setBlurStrength(e.target.value);
+    console.log('Updated to blurStrength:' + e.target.value);
+    const newBlurStrength = parseInt(e.target.value);
+    onBlurStrengthChanged(newBlurStrength);
+  }
+
+  return (
+    meetingStatus !== MeetingStatus.Succeeded ?
+    <MeetingForm /> :
+    <>
+      <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
```

Comment:
Shall we use `width: '100%'` instead to keep the style consistent with roster test app?

Reply:
yup - thanks

Reply:
I think you updated the wrong place.
Could you please update:
* line 37 should be `width: '70%'`
* line 93 should be `width: '100%'`

https://github.com/aws/amazon-chime-sdk-component-library-react/blob/52b49880fb7a3285daea7b5eabc546080d3877f8/integration/app/test-demo/src/pages/VideoFilterTestApp.tsx#L36-L46

https://github.com/aws/amazon-chime-sdk-component-library-react/blob/52b49880fb7a3285daea7b5eabc546080d3877f8/integration/app/test-demo/src/pages/VideoFilterTestApp.tsx#L87-L116

Reply:
I see - it looks visually the same but this is fine too.

Reply:
The style of `ControlBar` is same but the width of `MeetingForm` is the entire screen.

Reply:
Yeah, maybe it might be better for smaller screens to have wider meeting form. done